.github/workflows/lint-test-js.yml

```diff
@@ -12,7 +12,7 @@ jobs:
             - name: Check out repository code
               uses: actions/checkout@v2
             - name: Setup Node.js
-              uses: actions/setup-node@v2-beta
+              uses: actions/setup-node@v2.1.4
```

Comment:
There were some issues when doing npm install that were related to the beta version of setup-node. upgrading it to 2.x stable has fixed it.